blocks/explore-articles/explore-articles.js

```diff
@@ -49,59 +52,39 @@ const buildSelect = (type, array, text) => {
   return select;
 };
 
-const buildArticle = (e) => {
-  const article = createElement('div', { classes: ['article'] });
-  const articleImage = createElement('div', { classes: 'article-image' });
-  const articleContent = createElement('div', { classes: 'article-content' });
+const createTruckSection = (trucks) => {
+  const truckList = trucks?.map((item) => item.replace('Mack ', '')).join(', ');
+  return `
+    <div class="article-truck">
+      <img
+        class="truck-icon"
+        src="/icons/Truck_Key_icon.svg"
+        alt="truck icon"
+      />
+      <p class="article-truck-text">${truckList}</p>
+    </div>`;
+};
 
+const buildArticle = (e, idx) => {
   const linkUrl = new URL(e.path, window.location.origin);
-
-  const categoriesWithDash = e.category.replaceAll(' ', '-').toLowerCase();
+  const categoriesWithDash = e.category;
```

Comment:
I'd rename `categoriesWithDash` if we don't add dashes anymore with `replaceAll(' ', '-').toLowerCase()`.

Reply:
no, this is still needed, for some reason I deleted it 🤷